pkg/stream/producer_daemon.go

```diff
@@ -31,23 +31,23 @@ var (
 )
 
 type ProducerDaemonSettings struct {
-	Enabled bool `cfg:"enabled" default:"false"`
+	Enabled bool `cfg:"enabled"                default:"false"`
```

Comment:
Why are we inserting random spaces here? I don't think it makes sense to align them when nothing else is aligned because of the comments

Reply:
I guess that was auto aligned that way because of a missing new line after each property before the next comment, so it was recognized as one block of properties. Fixed that by adding new lines.